_posts/necessities/2018-02-05-chile-oil.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: Chile Oil
+category: necessities
+---
+
+I always have chile oil in a jar in the pantry. My mother makes her own, and my recipe is an adaptation of hers. 
+
+- Start with whole, dried chilis. 
```

Comment:
Given the paragraphs, do you want to actually keep this as a list? You could remove the bullets.

Reply:
I thought you liked lists